File: 1-codes/micro/micro05.py
def micro05():
    x,m,h = 0,0,0
    nome,sexo = "",""
    for x in range(5):
        print("Digite o nome: ")
        nome = input()
        print("H - Homem ou M - Mulher")
        sexo = input()
        if sexo == 'H':
            h = h+1
        elif sexo == 'M':
            m = m+1
        else:
            print("Sexo so pode ser H ou M!")

    print("Foram inseridos " + str(h) + " Homens")
    print("Foram inseridas " + str(m) + " Mulheres")

micro05()

File: 1-codes/micro/micro05.c
#import<stdio.h>
#include <string.h>

int main(int argc, char *argv[]){
  int x, m, h;
  char nome, sexo;

  for (x = 0; x < 5; x++) {
    printf("Digite o nome: ");
    scanf("%c", &nome);

    printf("H - Homem ou M - Mulher: ");
    scanf("%c", &sexo);

    switch (sexo) {
      case 'H':
        h = h + 1;
        break;
      case 'M':
        m = m + 1;
        break;
      default:
        printf("Sexo só pode ser H ou M!");
        break;
    }
  }

  printf("Foram inseridos %d Homens", h);
  printf("Foram inseridas %d Mulheres", m);

	return 0;
}
